doc/source/conf.py

```diff
@@ -135,6 +135,7 @@ def _stop_fluent_container(gallery_conf, fname):
             .split()
         )
         for container_name in container_names:
+            print(f"Container still running for script {fname}")
```

Comment:
Prints example script that leaves stale containers

Reply:
I think in some test cases we are not exiting sessions explicitly by using `session.exit()`, because for similar kind of issue we have updated `test_fluent_containes` where we have called `exit()` explicitly.

Reply:
Sure.